Amazon.QLDB.Driver/driver/QldbDriverBase.cs

```diff
@@ -110,7 +208,7 @@ internal void ReleaseSession(T session)
                     "Errored Transaction ID: {}. Error cause: {}",
                     qte.TransactionId,
                     qte.InnerException.ToString());
-                replaceDeadSession = !qte.IsSessionAlive;
+                bool replaceDeadSession = !qte.IsSessionAlive;
```

Comment:
Nit: I think we can probably get rid of the replaceDeadSession local variable here, unless you think it brings readability to the code.

Reply:
Done.